rbi/stdlib/open3.rbi

```diff
@@ -99,7 +99,7 @@ module Open3
   sig do
     params(
       cmd: T.any(String, T::Array[String]),
-      opts: T::Hash[Symbol, T.untyped],
```

Comment:
This required that every keyword arg be provided a hash

```ruby
f(x: {})
```

instead of allowing any keyword args

```ruby
f(x: '')
```

These errors were all previously silenced because of #6169 (it's common to call these methods with splatted positional arguments).